payjoin/src/receive/v2/mod.rs

```diff
@@ -146,6 +175,79 @@ impl<State: ReceiverState> core::ops::DerefMut for Receiver<State> {
     fn deref_mut(&mut self) -> &mut Self::Target { &mut self.state }
 }
 
+/// Extract an OHTTP Encapsulated HTTP POST request to return
+/// a Receiver Error Response
+pub fn extract_err_req(
+    err: &JsonReply,
+    ohttp_relay: impl IntoUrl,
+    session_context: &SessionContext,
+) -> Result<(Request, ohttp::ClientResponse), SessionError> {
+    let subdir = subdir(&session_context.directory, &session_context.id());
```

Comment:
master does a expiry check that is missing here
https://github.com/spacebear21/rust-payjoin/blob/69755571a28f1b8773e8bc4882a36a5528aef251/payjoin/src/receive/v2/mod.rs